### ISSUE TITLE
Django-sqrl repo

### ISSUE THREAD

**Issue by tatawilly** [miki725/django-sqrl#6]
Cannot reproduce install instructions.  For example 'pip install django-sqrl' does nothing.  Having cloned the repo, would like to exercise the server in a local environment. How does one transform the repo into a django mandated state?   Howw is the repo translated into a django project?     Thanks...

**Comment by miki725** [comment]
its not on PyPI yet. its still in development. Im not following your other questions. Can you elaborate?

**Comment by njf44** [comment]
Has there been any progress getting this into an available repository?

**Comment by miki725** [comment]
no. Im waiting for the protocol to settle to sync up the implementation here to the final protocol version and also pretty busy at work so some of other projects are on side-burn for a while

**Comment by bschollnick** [comment]
I'm sorry, I've just spent the last 45 minutes trying to find a viable SQRL Python 3 / Django 3 combination, and failing.

And yes, I know that Steve didn't "Finalize" the code until a month or two ago, but the last Github update is from May, 2015.  
Is this code going to be updated to Python 3 / Django 3?
